packages/angular/ssr/src/routes/ng-routes.ts

```diff
@@ -530,9 +554,9 @@ function buildServerConfigRouteTree({ routes, appShellRoute }: ServerRoutesConfi
       continue;
     }
 
-    if (path.includes('*') && 'getPrerenderParams' in metadata) {
+    if ('getPrerenderParams' in metadata && !path.endsWith('/**') && path.includes('*')) {
```

Comment:
Bug: Does this handle `/foo/*/bar/**` correctly? I think it would pass but should emit an error?